src/Entity/User.php

```diff
@@ -121,6 +121,9 @@ public function __construct()
     #[ORM\Column(type: Types::STRING, length: 39, nullable: true)]
     private string $ipAddress;
 
+    #[ORM\Column(type: Types::BOOLEAN, options: ['default' => 0])]
```

Comment:
Add nullable: false, it should not be null

Reply:
Done